web-client/src/components/data/dataset_input/FileSelection.vue

```diff
@@ -92,7 +92,9 @@
 
         <!-- If no preview of the selected file, display the nbr. of uploaded files -->
         <div class="pt-8 flex flex-col items-center pb-2">
-          <div class="mb-4 flex justify-center items-center text-center md:text-left sm:text-lg text-disco-blue">
+          <div 
+            v-if="selectedFiles?.length"
```

Comment:
[explicit is better than implicit.](https://peps.python.org/pep-0020/)

```suggestion
            v-if="selectedFiles?.length ?? 0 > 0"
```